scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/services/GraphQLMerlinService.java

```diff
@@ -879,6 +929,75 @@ public Optional<SimulationResults> getSimulationResults(PlanMetadata planMetadat
     }
   }
 
+  public record DatasetMetadata(DatasetId datasetId, Duration offsetFromPlanStart){};
```

Comment:
Minor, can ignore: Can all the records declared in this class be gathered into one section at the top of this file for discoverability?